skbeam/core/accumulators/tests/test_binned_statistic.py

```diff
@@ -23,51 +24,112 @@ def setup(self):
         colarr = np.arange(colsize)
         rowgrid, colgrid = np.meshgrid(rowarr, colarr, indexing='ij')
         self.rgrid = np.sqrt(rowgrid**2 + colgrid**2)
+        self.phigrid = np.arctan2(colgrid, rowgrid)
```

Comment:
as far as I can tell, it seems to be just a numerical precision error (I checked the arrays that resulted during the test), and it seems to be a difference in the 16th decimal digit or so. For example, replacing the test with:
```assert_array_almost_equal(ref, binned,decimal=10)```
will pass.

So under the hood, I would be tempted to say that the calculation is not exactly the same in terms of numerical operations, but the general API is not broken and what we expect it to do has not changed.
So I think one option could be to  leave the code as is and use `assert_array_almost_equal` in the test. What do you think?